README.md

```diff
@@ -13,16 +13,16 @@ This project is a thin wrapper that exposes API Spec-related capabilities from [
 | **Installation** | [NPM](#javascript) | [Gradle/Maven](#java) |
 | **Object-oriented interface** | ["WebApi" Model](https://raml-org.github.io/webapi-parser/js/classes/_webapi_parser_.webapibaseunit.html) | ["WebApi" Model](https://raml-org.github.io/webapi-parser/js/classes/_webapi_parser_.webapibaseunit.html) |
 | **Package** | [![NPMJS](https://img.shields.io/npm/v/webapi-parser.svg?style=flat)](https://www.npmjs.com/package/webapi-parser) | [![Maven Central](https://img.shields.io/static/v1.svg?style=flat&logo=java&label=%20&labelColor=white&labelColor=007396&color=007396&message=Maven%20Central)](https://search.maven.org/artifact/org.raml/webapi-parser/) |
-| **Examples** | [JavaScript examples](./examples/js/README.md) | [Java examples](./examples/java/README.md) |
+| **Examples** | [JavaScript examples](https://github.com/raml-org/webapi-parser/tree/master/examples/js/README.md) | [Java examples](https://github.com/raml-org/webapi-parser/tree/master/examples/java/README.md) |
```

Comment:
why not keep the links relative?

Reply:
Because this readme is copied to gitbook and is used as its index page. And on gitbook relative links wont work because there they would be relative to the github pages hosting address and not repo.

Reply:
Let's NOT copy the README then and use the SUMMARY instead. Good enough for now. Pushing a fix...